News-and-Events-Titleの作成

Yanmannu opened the pull request:
## セルフチェック
- [ ○] お手本通りの見た目であるか　ホバー時の挙動、レスポンシブ対応など要確認
- [ ○] ブランチ名、ファイル名、クラス名、インデントなど、コーディング規則を逸脱していないか
- [ ○] 記述の重複はないか

リンク
[https://60caedd1c637e00039f4d86a-ucidouckac.chromatic.com/](chromatic上の該当ページへのリンクを貼る)

Commit:
News-and-Events-Titleの作成

Yanmannu committed:
Update news-and-events-title.html

Commit:
見出しタグの修正

Yanmannu commented:
見出しタグが違うことに気づき、直しました。

Chromaticのリンク
https://60caedd1c637e00039f4d86a-crjbsnxncm.chromatic.com/

haruchu commented:
こちらのpaddingは必要ですか？
あくまでパーツを作っており、後々のスタイル崩れにもつながるので綺麗にまとめましょう！
<img width="812" alt="スクリーンショット 2022-02-17 16 40 25" src="https://user-images.githubusercontent.com/66406382/154428232-3249adf0-5d68-46a1-b5a7-b385e632e046.png">

haruchu commented:
このmarginも必要ありません
<img width="270" alt="スクリーンショット 2022-02-17 16 41 42" src="https://user-images.githubusercontent.com/66406382/154428437-54f002bf-3e4b-47ee-bdfc-9ddd3445bc74.png">

haruchu commented on src/components/molecules/news-and-events-title/news-and-events-title.html:
```diff
@@ -0,0 +1,2 @@
+<h2 class="news-and-events-title-En">News</h2>
+<h2 class="news-and-events-title-Ja">お知らせ</h2>
```
どちらもh2タグはおかしいですね。
hタグについて調べてみましょう。
また、これらをdivタグのようなものでひとまとめにしておくと良いと思います。
1つのパーツを作っているので。

Yanmannu committed:
クラスの変更

Yanmannu committed:
marginの修正

一つ前の課題ではmarginをつけてとのことでしたが、marginをつけるというのは、部品と部品の間のmarginであって、基本的には周りにはいらないという認識でよろしいでしょうか

haruchu commented on src/components/molecules/news-and-events-title/news-and-events-title.html:
```diff
@@ -0,0 +1,4 @@
+<div class="news-and-events-title">
+<h2 class="news-and-events-title-En">News</h2>
+<div class="news-and-events-title-Ja">お知らせ</div>
```
divタグはおかしいですね。
spanタグとかはどうでしょうか？調べてみてください。

haruchu commented on src/components/molecules/news-and-events-title/news-and-events-title.scss:
```diff
@@ -0,0 +1,24 @@
+  }
+  &-Ja {
+    font-family: $fontfamily-gothic;
+    padding-top: 3px;
```
この使い方ですとpaddingは適していないかと思います。

haruchu commented on src/components/molecules/news-and-events-title/news-and-events-title.scss:
```diff
@@ -0,0 +1,24 @@
+
+    font-size: $fontsize-m;
+    @media screen and (min-width: $grid-breakpoints-lg) {
+      padding-top: 15px;
```
同様

haruchu reviewed (commented):
コメントしました

Yanmannu committed:
spanタグに変更

Yanmannu committed:
divで囲み、部品化。参考サイトをお手本に作成

haruchu commented on src/components/molecules/news-and-events-title/news-and-events-title.html:
```diff
@@ -0,0 +1,4 @@
+<div class="news-and-events-title">
+<h2 class="news-and-events-title-En">News</h2>
+<span class="news-and-events-title-Ja">お知らせ</span>
```
子要素はインデントしましょう！

haruchu commented on src/components/molecules/news-and-events-title/news-and-events-title.scss:
```diff
@@ -0,0 +1,21 @@
+    margin-left: 16px;
+    font-size: $fontsize-m;
+    @media screen and (min-width: $grid-breakpoints-lg) {
+    }
```
何もしていないのであれば消しておきましょう。

haruchu reviewed (commented):
細かいですが、コメントしました。

Yanmannu committed:
Update news-and-events-title.scss

Yanmannu committed:
Update news-and-events-title.html

Yanmannu committed:
Delete news-and-events-title.css

Yanmannu committed:
Delete news-and-events-title.css.map

Yuto-tus commented on src/components/molecules/news-and-events-title/news-and-events-title.scss:
```diff
@@ -0,0 +1,19 @@
+    }
+  }
+  &-Ja {
+    font-family: $fontfamily-gothic;
```
この指定は以前にした指定と被っていると思われます。
確認してみてください。

Yanmannu committed:
Update news-and-events-title.scss
